modules/ROOT/pages/compression/compression-module.adoc

```diff
@@ -136,28 +130,23 @@ fail with a `COMPRESSION:TOO_MANY_ENTRIES` error, because it will not be able
 to choose only one entry to return. For these cases, you should use the
 Extract operation.
 
+== Archive and Extract a Multiple Entries
```

Comment:
I wonder if this should be "Archive and Extract Multiple Entries" or perhaps "Archive and Extract to Multiple Entries".